SpinCore_pp/run_field_dep_justMw.py

```diff
@@ -118,125 +118,125 @@ def verifyParams():
-                            ('marker','start',nScans),
-                            ('phase_reset',1),
-                            ('delay_TTL',deblank),
-                            ('pulse_TTL',p90,0.0),
-                            ('delay',tau),
-                            ('delay_TTL',deblank),
-                            ('pulse_TTL',2.0*p90,0.0),
-                            ('delay',deadtime),
-                            ('acquire',acq_time),
-                            ('delay',repetition),
-                            ('jumpto','start')
-                            ])
-                        #}}}
-                    print("\nSTOPPING PROG BOARD...\n")
-                    SpinCore_pp.stop_ppg();
-                    print("\nRUNNING BOARD...\n")
-                    SpinCore_pp.runBoard();
-                    raw_data = SpinCore_pp.getData(data_length, nPoints, nEchoes, nPhaseSteps, output_name)
-                    raw_data.astype(float)
-                    data_array = []
-                    data_array[::] = complex128(raw_data[0::2]+1j*raw_data[1::2])
-                    print("COMPLEX DATA ARRAY LENGTH:",shape(data_array)[0])
-                    print("RAW DATA ARRAY LENGTH:",shape(raw_data)[0])
-                    dataPoints = float(np.shape(data_array)[0])
-                    if x == 0 and B0_index == 0:
-                        time_axis = np.linspace(0.0,nEchoes*nPhaseSteps*acq_time*1e-3,dataPoints)
-                        data = ndshape([len(data_array),nScans,len(field_axis),1],['t','nScans','Field','power']).alloc(dtype=np.complex128)
-                        data.setaxis('t',time_axis).set_units('t','s')
-                        data.setaxis('nScans',r_[0:nScans])
-                        data.setaxis('Field',field_axis)
-                        data.setaxis('power',r_[powers])
-                        data.name(node_name)
-                        data.set_prop('acq_params',acq_params)
-                    data['nScans',x]['Field',B0_index]['power',0] = data_array
-                last_power = this_power
-            SpinCore_pp.stopBoard();
-            print("FINISHED B0 INDEX %d..."%B0_index)
-            print("\n*** *** ***\n")
+                        next_power += 3
+                b.set_power(this_power)
+            else:
+                b.set_power(this_power)
+            time.sleep(15)
+            with prologix_connection() as p:
+                with gigatronics(prologix_instance=p, address=7) as g:
+                    meter_powers = g.read_power()
+                    print("POWER READING",meter_powers)
+        for B0_index,desired_B0 in enumerate(field_axis):
+            true_B0 = x_server.set_field(desired_B0)
+            print("My field in G is %f"%true_B0)
+            time.sleep(3.0)
+            carrierFreq_MHz = gamma_eff*true_B0
+            print("My frequency in MHz is",carrierFreq_MHz)
+            acq_params['carrierFreq_MHz'] = carrierFreq_MHz
```

Comment:
OK -- what I say there isn't using quite the right words -- rather, you want to pull these variables automatically from the global namespace -- that's done [here](https://github.com/jmfrancklab/spincore_apps/pull/8/files#r769073960)

in general, you should not be messing directly with `acq_params['...']`, since the code I link to there does it for you.

Reply:
the code hasn't changed here.

Reply:
oops changed run_field_dep_justMw.py inside examples rather than this one inside SpinCore_pp but it should be good now!